xclim/indices/_conversion.py

```diff
@@ -2017,3 +2019,157 @@ def mean_radiant_temperature(
         0.25,
     )
     return mrt.assign_attrs({"units": "K"})
+
+
+@declare_units(wind_speed="[speed]", h="[length]", h_r="[length]")
+def wind_profile(
+    wind_speed: xr.DataArray,
+    h: Quantified,
+    h_r: Quantified,
+    method: str = "power_law",
+    **kwds,
+):
+    r"""Wind speed at a given height estimated from the wind speed at a reference height.
+
+    Estimate the wind speed based on a power law profile relating wind speed to height above the surface.
+
+    Parameters
+    ----------
+    wind_speed : xarray.DataArray
+        Wind speed at the reference height.
+    h : Quantified
+        Height at which to compute the wind speed.
+    h_r : Quantified
+        Reference height.
+    method : {"power_law"}
+        Method to use. Currently only "power_law" is implemented.
+    kwds : dict
+        Additional keyword arguments to pass to the method. For `power_law`, this is `alpha`, which takes a default
```

Comment:
Could add some additional clarification, if desired: "For `power_law`, this is `alpha`, which takes a default value of 1/7, but is highly variable based on topography, surface cover and atmospheric stability. "